src/mame/sega/segas32_v.cpp

```diff
@@ -646,20 +643,19 @@ int segas32_state::compute_clipping_extents(screen_device &screen, int enable, i
 
 void segas32_state::compute_tilemap_flips(int bgnum, int &flipx, int &flipy)
 {
-	/* determine if we're flipped */
-	int global_flip = (m_videoram[0x1ff00 / 2] >> 9)&1;
-
-	flipx = global_flip;
-	flipy = global_flip;
+	/* determine flip bits */
+	int global_flip    = (system32_videoram[0x1ff00 / 2] >> 9) & 1;
+	int layer_flip     = (system32_videoram[0x1ff00 / 2] >> bgnum) & 1;
+	int prohibit_flipy = (system32_videoram[0x1ff00 / 2] >> 8) & 1;
```

Comment:
For new or replaced code that does (x >> y & 1), we more commonly use BIT(x, y).
And use // for single line comments btw.

Reply:
`= BIT(system32_videoram[0x1ff00 / 2], 8);`

Reply:
BIT:
something >> shiftamount & 1
is equal to: BIT(something, shiftamount)

optionally, the 3rd param of BIT() is number of bits to mask, but I find that one harder to read.